app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/GermanKeyboard.java

```diff
@@ -63,4 +63,9 @@ public Locale getLocale() {
     public String getSpaceKeyText(String aComposingText) {
         return StringUtils.getStringByLocale(mContext, R.string.settings_language_german, getLocale());
     }
+
+    @Override
+    public String[] getDomains(String... domains) {
+        return super.getDomains(".de");
```

Comment:
German keyboard would also be used in Austria and Switzerland, so this should have included `.at` and `.ch`.

Reply:
Thanks @philip-lamb Added [here](https://github.com/MozillaReality/FirefoxReality/issues/2132).